### DISCUSSION TITLE
Missing .gitignore

### DISCUSSION THREAD

**Discussion by jeotek** [Lenti84/VdMot_Controller#117] [General]
Hi,
Found this projekt and it is exactly what I has been looking for.
So thanks for this and if I feel that I maybe can contribute.

There is something that I want to be added to the project as that is ".DS_Store" in .gitignore due to the fact that I do my work on MACOS and this file is annoying to have in a repository.

**Comment by Lenti84** [comment]
Hi Jeotek,
I recently created .gitignore in developer branch. Please try out if this is what helps Mac OS users.

**Reply by jeotek** [reply]
Thanks, it helps.
Maybe change the comment to "## MACOS specific ########"
For better control in the future, of what is what.
